### DISCUSSION TITLE
Have trouble downloading secretflow after "pip install -U secretflow", what seems to be the problem? How may I solve this? Thanks!

### DISCUSSION THREAD

**Discussion by ryuuTAylor** [secretflow/secretflow#649] [Q&A]
<img width="1440" alt="Screenshot 2023-06-09 at 3 00 02 PM" src="https://github.com/secretflow/secretflow/assets/113742295/1909fbc5-4c45-4fdb-806a-48cb73484be5">

**Comment by 6fj** [comment]
Hi @ryuuTAylor 

看上去你应该是在mac上安装secretflow吧？按照错误提示，请先按照提示安装cmake：https://cmake.org/download/

之后请在github issue中提问，感谢！

**Reply by ryuuTAylor** [reply]
<img width="1440" alt="Screenshot 2023-06-10 at 9 31 58 PM" src="https://github.com/secretflow/secretflow/assets/113742295/5d0c327d-8116-4442-a593-f116c19886ea">
<img width="662" alt="Screenshot 2023-06-10 at 9 34 56 PM" src="https://github.com/secretflow/secretflow/assets/113742295/22e5a956-bac4-4156-9c31-0f7011ff3c4a">
非常感谢，我下载了cmake，但是不太会用cmake，error message也改变了，我缺少的这个text file是怎么回事呢？我是根据安装流程下载的，遇到了这个问题，还需要下载什么包之类的吗？

**Comment by Chrisdehe** [comment]
hi,@ryuuTAylor 
我们后续在这条issue中进行处理好吗？
https://github.com/secretflow/secretflow/issues/650

**Reply by WakumaAyanaJifar** [reply]
I am facing exactly the same problem. I tried all method you guys mentioned to install secretflow. But none of them worked for me. Please could you helps pls
